src/ModelContextProtocol/Server/AIFunctionMcpServerTool.cs

```diff
@@ -193,6 +193,30 @@ options.OpenWorld is not null ||
         return newOptions;
     }
 
+    private static CallToolResponse ConvertAiContentEnumerableToCallToolResponse(IEnumerable<AIContent> contentItems)
```

Comment:
Two nits:
1. AiContent => AIContent
2. Please move this method down to the end of the file, where it'll be closer to where it's consumed.

Otherwise, the change LGTM. Thanks.